xaitk_saliency/impls/gen_image_classifier_blackbox_sal/slidingwindow.py

```diff
@@ -0,0 +1,47 @@
+from typing import Tuple, Dict, Any
+
+import numpy as np
+from smqtk_classifier import ClassifyImage
+
+from xaitk_saliency.impls.gen_classifier_conf_sal.occlusion_scoring import OcclusionScoring
+from xaitk_saliency.impls.gen_image_classifier_blackbox_sal.occlusion_based import PerturbationOcclusion
+from xaitk_saliency.impls.perturb_image.sliding_window import SlidingWindow
+from xaitk_saliency.interfaces.gen_image_classifier_blackbox_sal import GenerateImageClassifierBlackboxSaliency
+
+
+class SlidingWindowStack (GenerateImageClassifierBlackboxSaliency):
+    """
+    Encapsulation of the perturbation-occlusion method using specifically
+    sliding windows and the occlusion-scoring method.
+    """
```

Comment:
In our corresponding RISEStack method, we duplicated the docstring with information on the input parameters. However, I see that's this is missing here (e.g. info on window_size, stride, threads).

Reply:
I indeed missed getting the parameter docs here. I will add them.